### DISCUSSION TITLE
Possible error in observation field of net solar forcing (FSNT)

### DISCUSSION THREAD

**Discussion by oyvindseland** [NCAR/ADF#328] [Q&A] [answered]
Hi 

The observational data for FSNT looks strange with slightly higher values in the polar regions than in the tropics. E.g 
https://ns2345k.web.sigma2.no/datalake/diagnostics/ADF/plots/n1850.ne30_tn14.hybrid.20240822_1_10_vs_Obs/website/html_img/plot_page_FSNT_ANN_Zonal_Mean.html

If anything my guess would be that it shows outgoing solar, not net. Are there anyone else who have looked at this variable.

**Comment by justin-richling** [comment, marked as answer]
Thanks for finding this @oyvindseland! You are correct, the variable the ADF was calling was just the outgoing shortwave. Now the obs file has the variable `fsnt` and the variable defaults yaml file has been updated to point to that variable.
